supreme_court_predictions/statistics/descriptives.py

```diff
@@ -0,0 +1,246 @@
+"""
+This file houses the class that is used to provide descriptive summary 
+statistics for the convokit datasets.
+"""
+import os
+
+import numpy as np
+import pandas as pd
+
+from supreme_court_predictions.util.contants import ENCODING_UTF_8_SIG
+
+from .datacleaner import DataCleaner
+
+
+class Descriptives:
```

Comment:
Could you also put the calling of this class in the `/supreme-court-ml-predictions/supreme_court_predictions/statistics/services.py` file?

Reply:
sure can!